assets/javascripts/discourse/components/explorer-schema/enum-info.js

```diff
@@ -0,0 +1,15 @@
+import Component from "@glimmer/component";
+
+export default class ParamInput extends Component {
```

Comment:
```suggestion
export default class EnumInfo extends Component {
```

(or change the file name 😛)